nncf/quantization/algorithms/fast_bias_correction/torch_backend.py

```diff
@@ -0,0 +1,144 @@
+from nncf.quantization.algorithms.fast_bias_correction.backend import FastBiasCorrectionAlgoBackend
+from nncf.torch.graph.node_utils import is_node_with_bias
+from nncf.torch.graph.operator_metatypes import PT_OPERATOR_METATYPES
+from nncf.torch.graph.transformations.command_creation import create_bias_correction_command
+from nncf.torch.graph.transformations.commands import PTBiasCorrectionCommand
+from nncf.torch.graph.transformations.commands import PTModelExtractionCommand
+from nncf.torch.graph.transformations.commands import PTTargetPoint
+from nncf.torch.quantization.layers import BaseQuantizer
+from nncf.torch.tensor import PTNNCFTensor
+from nncf.torch.tensor_statistics.collectors import PTMeanStatisticCollector
+from nncf.torch.tensor_statistics.collectors import PTNNCFCollectorTensorProcessor
+
+
+@ALGO_BACKENDS.register(BackendType.TORCH)
+class PTFastBiasCorrectionAlgoBackend(FastBiasCorrectionAlgoBackend):
+
+    @property
+    def operation_metatypes(self) -> Registry:
+        return PT_OPERATOR_METATYPES
+
+    @property
+    def tensor_processor(self) -> PTNNCFCollectorTensorProcessor:
+        return PTNNCFCollectorTensorProcessor()
+
+    TARGET_TYPE_TO_PT_INS_TYPE_MAP = {
+        TargetType.PRE_LAYER_OPERATION: TargetType.OPERATOR_PRE_HOOK,
+        TargetType.POST_LAYER_OPERATION: TargetType.OPERATOR_POST_HOOK,
+    }
+
+    @staticmethod
+    def target_point(target_type: TargetType,
+                     target_node_name: str,
+                     port_id: int) -> PTTargetPoint:
+        if NNCFGraphNodeType.INPUT_NODE in target_node_name or\
+            target_type == TargetType.POST_LAYER_OPERATION:
+            port_id = None
+        if target_type in PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP:
+            target_type = PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP[target_type]
+        return PTTargetPoint(target_type, target_node_name, input_port_id=port_id)
+
+    @staticmethod
+    def create_bias_correction_command(node: NNCFNode,
+                                       bias_value: np.ndarray,
+                                       nncf_graph: NNCFGraph) -> PTBiasCorrectionCommand:
+        return create_bias_correction_command(node, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> PTModelExtractionCommand:
+        return PTModelExtractionCommand(inputs[0])
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape: ReductionShape,
+                                 num_samples: Optional[int] = None,
+                                 window_size: Optional[int] = None) -> PTMeanStatisticCollector:
+        return PTMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_sub_input_output_names(subgraph: nn.Module) -> Tuple[str, str]:
+        # Pytorch does not have name for extracted node
```

Comment:
Are the names of the nodes in the new sub-graph match with the same that was provided to the `model_extraction_command`?

Reply:
Is not sub-graph. No names of node and no node, no graph in PT model.